zio-kafka/src/main/scala/zio/kafka/producer/TransactionalProducer.scala

```diff
@@ -22,6 +22,10 @@ trait TransactionalProducer {
 object TransactionalProducer {
   case object UserInitiatedAbort
   final case class TransactionLeaked(offsetBatch: OffsetBatch) extends RuntimeException
+  case object RebalanceSafeConsumerRequired
```

Comment:
This name suggest the consumer is safe in rebalances. This is unfortunately not the case, not can it ever be.
```suggestion
  case object RebalanceSafeCommitsRequired
```

Reply:
done